src/pages/index.tsx

```diff
@@ -38,7 +38,7 @@ export default function Home() {
         <Star className="absolute right-80    top-40" />
```

Comment:
New logo is missing in header. Design assets were posted by Zepti in the Discord group.

Reply:
Creating new issue instead #120